pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py

```diff
@@ -57,7 +57,7 @@ def process(self, context):
                         instance.data["families"]
                     )
                 )[0]
-                instance.data["outputPath"] = os.path.join(
+                instance.data["output"] = os.path.join(
```

Comment:
To me, the original name was more indicative of what it represented. Output could be any variable type - path would suggest a string. I'd lose the camelcasing though if anything.